lib/pbench/test/unit/server/test_indexer.py

```diff
@@ -22,12 +22,33 @@ def test_not_found():
 
     @staticmethod
     def test_fallbacks():
-        keyvals = {"name": "mybm"}
+        templ = "%benchmark_name%_UID"
+        res = ResultData.expand_uid_template(templ, dict(foo=1, bar=2))
+        assert res == templ
+
+        keyvals = {"benchmark_name": "mybm"}
         templ = "%benchmark_name%_UID"
         res = ResultData.expand_uid_template(templ, keyvals)
         assert res == "mybm_UID"
```

Comment:
Not to quibble, but why the difference in idiom between these two test cases?  (I.e., the first specifies the dictionary in-line, while the second sets up a single-use local variable....)  Ditto for lines 46 and 51.

And, as long as I'm at it, why reassign `templ` every time if the value is the same in each case?  (Perhaps use a single assignment?  Perhaps just in-line the value?)